assets/js/payment-method-extensions/payment-methods/stripe/credit-card/payment-method.js

```diff
@@ -34,9 +34,9 @@ const CreditCardComponent = ( {
 	const { customerId } = billing;
 	const [ sourceId, setSourceId ] = useState( '' );
 	const stripe = useStripe();
-	const [ shouldSavePayment, setShouldSavePayment ] = useState(
-		customerId ? true : false
-	);
+	// Saving payment method is always opt-in (based on previous/shortcode
+	// behaviour); so default to false.
```

Comment:
I don't think we need this comment block. We don't need rationale in the code for every default decision. If you are leaving it to help guide reviews, one way I deal with that is I'll leave my _own_ review comments on a pull for reviewers to see (rather than comment directly in the code).